<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-N2O8J3T" failures="2" tests="9" name="Test" time="0.228" errors="0" timestamp="2024-06-25T18:00:30 IST">
  <testcase classname="listeners.AnalyzerClass" name="test1" time="0.026"/>
  <testcase classname="listeners.AnalyzerClass" name="test2" time="0.017">
    <skipped/>
  </testcase> <!-- test2 -->
  <testcase classname="listeners.AnalyzerClass" name="test2" time="0.006">
    <skipped/>
  </testcase> <!-- test2 -->
  <testcase classname="listeners.AnalyzerClass" name="test2" time="0.002">
    <skipped/>
  </testcase> <!-- test2 -->
  <testcase classname="listeners.AnalyzerClass" name="test2" time="0.007">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at listeners.AnalyzerClass.test2(AnalyzerClass.java:14)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
  <testcase classname="listeners.AnalyzerClass" name="test3" time="0.002">
    <skipped/>
  </testcase> <!-- test3 -->
  <testcase classname="listeners.AnalyzerClass" name="test3" time="0.003">
    <skipped/>
  </testcase> <!-- test3 -->
  <testcase classname="listeners.AnalyzerClass" name="test3" time="0.002">
    <skipped/>
  </testcase> <!-- test3 -->
  <testcase classname="listeners.AnalyzerClass" name="test3" time="0.008">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at listeners.AnalyzerClass.test3(AnalyzerClass.java:19)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
</testsuite> <!-- Test -->
